universal-application-tool-0.0.1/app/filters/ValidAccountFilter.java

```diff
@@ -36,6 +37,21 @@ public EssentialAction apply(EssentialAction next) {
                   Results.redirect(org.pac4j.play.routes.LogoutController.logout()));
             }
           }
+
+          // Check to see if the account is the unconfirmed email placeholder account from Seattle
+          // IDCS. If it is,
+          // redirect to a custom support page for that purpose.
+          if (profile.isPresent()
+              && profileUtils.accountIsIdcsPlaceholder(profile.get())
+              // Guard against a redirect loop
+              && !request
+                  .uri()
+                  .startsWith(routes.SupportController.handleUnconfirmedIdcsEmail().path())) {
```

Comment:
I presume this works since you screen shotted the page?

I wouldn't expect the request.uri() to only be a path

Reply:
Looks like uri is the path from looking at the code. this was also removed.